classes/tax/TaxManagerFactory.php

```diff
@@ -35,7 +35,7 @@ class TaxManagerFactoryCore
      * Returns a tax manager able to handle this address.
      *
      * @param Address $address
-     * @param string $type
+     * @param int $type
```

Comment:
[Line 46](https://github.com/PrestaShop/PrestaShop/pull/22547/files#diff-cfc7507489b1f047388a500c47d135e14029f9effce9bddc2c0851d1ffae4e52R46) `$type` is passed to `execHookTaxManagerFactory` that expect a `string` 🤔